packages/admin-next/dashboard/src/routes/users/user-edit/index.ts

```diff
@@ -1 +1 @@
-export { UserEdit as Component } from "./user-edit"
+export { UserEdit as Component } from "../../../modules/user/user-edit"
```

Comment:
q: whats this change for?

Reply:
We should strive to move all reusable (between v1 and v2) components to `modules` to reduce the maintenance burden. This change is here because I was able to share the user-edit between the two modes.